### DISCUSSION TITLE
高分辨率的显示器下轨迹显示会变卡

### DISCUSSION THREAD

**Discussion by silent0lzx** [shuax/MouseInc.Settings#148] [Q&A]
你好，作者大佬：
MouseInc真的很好用。
用了很长高分辨率的显示器，试过多个显示器后才发现，在高分辨率下，如2K屏，4K屏，右键的轨迹显示会一卡一卡的，观察了下CPU情况，也会出现明显的上升。DPI越高，程序越卡。
不知道大佬有没有解决方法，或者能不能对这方面进行优化？

**Comment by silent0lzx** [comment]
补充，也可能是在双显示器的环境下更明显

**Comment by shuax** [comment]
我自己只有2k，是不卡的。确实有很多4k反应卡，我没条件优化。双显示器就会叠加，所以会更卡。
